### ISSUE TITLE
Feature Request: Create TFENV_TF_VERSION

### ISSUE THREAD

**Issue by bytebeast** [tfutils/tfenv#405]
Feature Request: 

Create TFENV_TF_VERSION variable that has current selected version. So that I may use that in my prompt. 

Thanks.

**Comment by next-jesusmanuelnavarro** [comment]
Wouldn't `terraform --version` itself suffice? i.e.:
```sh
$ terraform --version --json | jq -r .terraform_version
1.5.5
$
```

Note that tfenv is meant to *"dynamically"* load Terraform, so it may happen that no terraform at all is activated at any given prompt, i.e.:
```sh
$ terraform --version --json | jq -r .terraform_version
Specifically asked for min-required via terraform{required_version}, but none found
$
```
You can even run into a prompt where not even tfenv is active, i.e.:
```sh
$ terraform --version --json | jq -r .terraform_version

Command 'terraform' not found
$
```
You'll have to cope with that case one way or the other.

**Comment by mengesb** [comment]
Maybe https://github.com/ohmyzsh/ohmyzsh/tree/master/plugins/terraform is what you're looking for?

**Comment by Zordrak** [comment]
Agreed with commenters - definitely not tfenv's scope to modify users global env.
